src/NuGet.Core/NuGet.Packaging/Signing/Archive/Crc32.cs

```diff
@@ -12,34 +14,36 @@ namespace NuGet.Packaging.Signing
     public static class Crc32
     {
         // Table of CRCs of all 8-bit messages.
-        private static long[] CrcLookUpTable = new long[256];
+        private static uint[] CrcLookUpTable = new uint[256];
 
         // Flag: has the table been computed? Initially false.
         private static bool CrcLookUptableComputed = false;
 
-        // Magic number used for CRC32
-        private const long Crc32Polynomial = 0xedb88320L;
+        // 0xedb88320 is the reversed form (least significant bit first) of the
+        // CRC-32 polynomial 0x04c11db7 (most significant bit first).
```

Comment:
How does this relate to the magic number in the pkware spec?

Reply:
Look at the [test](https://github.com/NuGet/NuGet.Client/pull/1885/files#diff-94ad5e71673efcacdd7d438a66f05c24R46).